builder/builder.go

```diff
@@ -31,7 +31,7 @@ type Source interface {
 	// delete the temporary directory in Close().
 	Close() error
 	// Hash returns a checksum for a file
-	Hash(path string) (string, error)
+	Hash(path string, targetOS string) (string, error)
```

Comment:
Instead of changing the interface I think it would be better to have the implementations of `Source()` store the `Platform`.

Reply:
@dnephin Turns out this needs to be done for LCOW as well, so it doesn't need to know the targetOS at all. These are just source files on the local machine which will be NTFS. Hence updated to be a much simpler fix.

Reply:
Ya that is a much smaller change!